src/components/simple-material/SimpleMaterial.tsx

```diff
@@ -73,6 +73,7 @@ const SimpleMaterial: FC<SimpleMaterialProps> = ({
       </div>
       <div className="simple-material__favourite">
         <ButtonFavourite
+          title={title[0]}
```

Comment:
Maybe I miss something but why does it work to use `title` both as an array and [cast it to a string](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/63682a68b7cd8d727ad170db7df2a52bfd8ce1d4/src/components/simple-material/SimpleMaterial.tsx#L84)

Reply:
I can try? is it always an array with only one entry?